src/components/standalone/Tilbakemelding/TilbakemeldingPopup.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState, FormEvent } from 'react';
+import styled from 'styled-components';
+import { Innholdstittel, Normaltekst, Element } from 'nav-frontend-typografi';
+import theme, { pxToRem } from '../../../styles/personOversiktTheme';
+import TilbakemeldingValg from './TilbakemeldingValg';
+import { Textarea } from 'nav-frontend-skjema';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { smilies } from './TilbakemeldingIkoner';
+import { useFocusOnMount } from '../../../utils/customHooks';
+
+const TilbakemeldingPopupContainer = styled.div`
+    ${theme.hvittPanel}
+    width: ${pxToRem(350)};
+    position: fixed;
+    right: ${pxToRem(25)};
+    bottom: ${pxToRem(100)};
+    z-index: 998;
+    padding: ${pxToRem(20)};
+    
+    &:focus {
+        ${theme.focus}
+    }
+`;
+const KnappWrapper = styled.div`
+    display: flex;
+    justify-content: space-between;
+`;
+const IkkeVis = styled.button`
+    background: none;
+    border: none;
+`;
+const TakkForTilbakemeldingenContainer = styled(TilbakemeldingPopupContainer)`
+    background-color: #98d0b0;
+    text-align: center;
+`;
+
+interface Props {
+    sporsmal: string;
+    erApen: boolean;
+    besvart: boolean;
+    settBesvart(tilfredshet: number, besvart: string): void;
+}
+
+interface InternalProps extends Props {
+    valgt: number;
+    kommentar: string;
+    settValgt(valgt: number): void;
+    settKommentar(kommentar: string): void;
+}
+
+function Kommentar(props: InternalProps) {
+    return (
+        <>
+            <Textarea
+                value={props.kommentar}
+                onChange={e => props.settKommentar((e as React.KeyboardEvent<HTMLTextAreaElement>).currentTarget.value)}
+                label="Si gjerne litt mer om opplevelsen av endringer."
+                maxLength={750}
+            />
+            <KnappWrapper>
+                <Hovedknapp role="submit">Send</Hovedknapp>
+                <IkkeVis
+                    role="button"
+                    className="lenke"
+                    onClick={event => {
+                        event.preventDefault();
+                        props.settBesvart(-1, '');
+                    }}
+                >
+                    Ikke vis denne igjen
+                </IkkeVis>
+            </KnappWrapper>
+        </>
+    );
+}
+
+function TilbakemeldingSkjema(props: InternalProps) {
+    if (props.besvart) {
+        return null;
+    }
+    const popup = React.createRef<HTMLDivElement>();
+    useFocusOnMount(popup);
+
+    const submitHandler = (event: FormEvent) => {
+        event.preventDefault();
+        props.settBesvart(props.valgt, props.kommentar);
+    };
+
+    return (
+        <TilbakemeldingPopupContainer ref={popup} tabIndex={-1}>
+            <Innholdstittel className="blokk-xs">Tilbakemelding</Innholdstittel>
+            <Normaltekst className="blokk-m">{props.sporsmal}</Normaltekst>
+            <form onSubmit={submitHandler}>
```

Comment:
er vel ikke brukt så mye `<form onSubmit={...}` i appen, men vet ikke hva som er best practise egentlig. Vi har vel kommet rundt at knapper trigger form'en ved å gi knappene `htmlType="button"` også knyttes submitt-metoden til onclick på knappen. Men vet ikke hva som er "best"

Reply:
Best practice er nok å bruke `onSubmit` tror jeg.
`onClick` på en spesifik knapp oppfører seg ikke helt som standard-forms ville gjort i gamle dager, e.g sånn som da PHP var kult. :P Mens `onSubmit` vil gi deg samme oppførsel som "native-web".

For å teste det kan man lage ett enkelt form med en `input[type=text]` og `button[type=submit]`. 
Ved `onSubmit` vil `enter` i tekst-feltet føre til en submit, mens det ikke vil skje om man bare har `onClick` på knappen.
Mener ihvertfall at det sånn ting fungerer. :p

Reply:
mhm, da har jeg egentlig ikke så sterke meninger. Husker at vi i noen tilfeller ønsket å forhindre at `enter` førte til submit, men det er vel ikke noe problem her.